modules/dag-l1/src/main/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustDbStorage.scala

```diff
@@ -0,0 +1,114 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.MonadCancelThrow
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import org.tessellation.dag.l1.domain.trust.storage.TrustStore
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustDbValues
+
+import doobie.implicits._
+import doobie.quill.DoobieContext
+import io.getquill.context.Context
+import io.getquill.{Literal, SqliteDialect}
+
+object TrustDbStorage {
+  case class StoredAddress(
+    peerId: String,
```

Comment:
It should be `PeerId`. See comments about nominal types. We have such type already in schema.

Reply:
Replaced the generic typed parameters with the values classes and added the appropriate decoders/encoders.